cerebrium/endpoints/async.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Asynchronous Execution (Preview)"
+description: "Execute calls to a Cerebrium app to be run asynchroously"
+---
+
+<Note>
+This is a new feature! As such, the API is still currently subject to changes.
+
+Most applications are expected to work with the current implementation.
+However, should you encounter an issue deploying a Custom Runtime please
+reach out to us on Discord!
+
+</Note>
+
+Many applications require asynchronous execution of functions in a "fire-and-forget" fashion
+in order to work optimally. In this scenario, you hand off execution of a function to be Cerebrium's
+responsibility, while you as the developer are responsible for ensuring that data is able leave the function.
+
+Through parameterizing your request to the Cerebrium API, you can enable your function to executed asynchronously
+by setting the `async` query parameter of your request to `true`. This would look something like this:
+
+```bash
+curl -X POST https://api.cortex.cerebrium.ai/v4/<YOUR-PROJECT-ID>/<YOUR-APP>/run?async=true'\
+       -H 'Content-Type: application/json'\
+       -H 'Authorization: Bearer <YOUR-JWT-TOKEN>\
+       --data '{"param": "hello world"}'
+```
+
+This would _immediately_ return a response akin to the following, with a body that only specifies a `run_id`:
+
+```bash
+HTTP/1.1 202 Accepted
+Connection: close
+Content-Length: 50
+Content-Type: application/json
+Date: Tue, 12 Nov 2024 03:13:25 GMT
+Server: envoy
+Vary: Origin
+X-Envoy-Upstream-Service-Time: 2
+X-Request-Id: 21eb3b98-4b10-9ad6-8681-a47172828024
+
+{"run_id":"21eb3b98-4b10-9ad6-8681-a47172828024"}
+```
+
+In the background, Cerebrium will now run this synchronous function for you. Note the term synchronous.
+Your function is still expected to behave **synchronously**. That is, it must open a connection, keep
```

Comment:
this was confusing to me in terms of it must keep the connection open? It makes me think:
- What does it mean to keep the connection open
- Can my function have the async definition or not?